### PR TITLE
feat(app): add sorting function for csv files

### PR DESCRIPTION
**Pull request by koji**
# Overview
add sorting function for csv files

[before]
Screenshot 2024-07-22 at 5 56 22 PM

[after]
https://github.com/user-attachments/assets/b14217cd-dd4e-4767-9773-a4ea93b0d6df

close AUTH-578

# Test Plan

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx [hunk 1]
```diff
@@ -57,6 +78,7 @@ export function ChooseCsvFile({
 
   const handleBackButton = (): void => {
     if (!isEqual(csvFileSelected, initialFileObject)) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```
Comment (jerader) [inline app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx]: this is suddenly needed?

Comment (koji) [reply app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx:81]: oh good catch

Comment (koji) [reply app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx]: on my end
```
Unsafe argument of type `any` assigned to a parameter of type `string`.eslint[@typescript-eslint/no-unsafe-argument](https://typescript-eslint.io/rules/no-unsafe-argument)
```

maybe as string would be better than eslint-disabled.

**Review by jerader — APPROVED** [review]
code lgtm!

**Pull request merged**